### DISCUSSION TITLE
Can't view stream

### DISCUSSION THREAD

**Discussion by GIBLechat** [GerryDazoo/Slinger#645] [General]
I cannot view a stream.  Output of slingplayer 4 below. Any thoughts to correct?  Thanks in Advance

Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
02/14/2024, 22:28:30.022   Streaming request from ('192.168.0.61', 50835)  /slingbox
STREAM=192.168.0.61:50835:0
8080 Using encrypted password: E1:[deleted]

Slinginfo  500 12 30 192.168.0.107 5201 3072 10 IVkuJJratWnRx9m
8080 Starting Stream for  192.168.0.61:50835
8080 Opening Control stream 0x2000 0 0
Connecting... ('192.168.0.107', 5201) Control
8080 Selecting Video Source 2
8080 Keycodes= [1, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 22, 24, 25, 27, 28, 29, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 46, 72, 73, 74, 75, 76, 77]
VideoParameters: Resolution= 12 FrameRate= 30 VideoBandwidth= 2000 VideoSmoothness= 63 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.0.107', 5201) Stream
Traceback (most recent call last):
  File "slingbox_server.py", line 836, in streamer
    s_ctl, stream  = start_slingbox_session(streams)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "slingbox_server.py", line 428, in start_slingbox_session
    first_buffer = bytearray(stream.recv(pksize, socket.MSG_PEEK))
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TimeoutError: timed out
8080 Badness starting slingbox session  timed out None
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

**Comment by GerryDazoo** [comment]
Try these:
1) Give your Slingbox a power cycle. (Not a factory reset)
2) Verify the video input to the box is working

**Reply by GIBLechat** [reply]
Gerry, thanks. Unfortunately, I am a couple thousand kilometers away from the Slingbox.  Is there any way to power cycle remotely?

Thanks.

**Reply by GIBLechat** [reply]
Next time I go to the house with the Slingbox, I will put a smart outlet on it so that I can turn off the power remotely. I should have thought of that before!

**Reply by GerryDazoo** [reply]
sorry no remote Slingbox power cycle.
